_drafts/react-example-todomvc.md

```diff
@@ -0,0 +1,129 @@
+---
+title: TodoMVC example, a React version
```

Comment:
I would prefer something like "TodoMVC: a reactive version". "Reactive" would refer to the more general concept of Reactive Programming, not to the name of a specific OCaml module.